### ISSUE TITLE
Question: More image encodings means more accuray?

### ISSUE THREAD

**Issue by sskhokhar** [ageitgey/face_recognition#895]
Hi,
I tried to build face recognition system using openCv but this lib looks more promising.
I saw an API to extract face encodings of an image which I can use to store it somewhere.

**My question is**, if I store more face encodings of **person A** (lets say 5 image encodings) and then I  recognize **person A** by providing stored 5 face encodings. 
=> Does it mean that more face-encodings of a person result in more accuray Or this lib can work with just 1 image encoding?

Regards,

**Comment by pedrohenriquebr** [comment]
Yes and no, KNN classifier has more accuracy by storing more faces. It's depending on your classification algorithm or classifier, so I suggest you to use SVM classifier man. 👍🏼

**Comment by SaddamBInSyed** [comment]
@pedrohenriquebr 
Is there any specific reason to use SVM instead of KNN?
In my case, I have around 50k images that need to be trained. 

I have choosed the below combo for each image,
knn(classifier) + cnn(model) + jitter=100. 

but I bit confused about which classifier will work best for my case. Knn or SVM

I am using Jetson Nano(gpu)
please advise,

**Comment by FedericoLanzani** [comment]
> @pedrohenriquebr
> Is there any specific reason to use SVM instead of KNN?
> In my case, I have around 50k images that need to be trained.
> 
> I have choosed the below combo for each image,
> knn(classifier) + cnn(model) + jitter=100.
> 
> but I bit confused about which classifier will work best for my case. Knn or SVM
> 
> I am using Jetson Nano(gpu)
> please advise,


Hi! The difference between svm and knn is, in a few words, this: https://github.com/ageitgey/face_recognition/issues/653#issuecomment-440952279
